aws/terraform/modules/metaflow/README.md

```diff
@@ -0,0 +1,17 @@
+# README
+
+This project is composed of modules which break up the responsibility into logical parts. See each module's 
+corresponding `README.md` for more details.
+
+Provides the core functionality for Metaflow which includes:
+
+- on demand processing (`computation`)
+- blob and tabular storage (`datastore`)
+- an API to record and query past executions (`metadata-service`)
+- orchestrated processing (`step-functions`)
+
+Depends on the output of the project `infra`.
```

Comment:
But there isn't really a hard dependency on `infra` specifically, right?. It is just this module needs a VPC and whatnot, and `infra` is an example of how to create those. 

If that ^ is correct, it maybe worth explaining in this doc a bit.

Reply:
That is absolutely correct, I'll update the documentation accordingly.